js-exercises/negative-index/negativeIndex.js

```diff
@@ -0,0 +1,30 @@
+const handler = {
+  get(target, prop) {
+    if (prop in target) {
+      return target[prop];
+    } else if (typeof prop !== "symbol") {
+      const index = Number(prop);
+      if (index < 0) {
+        return target[index + target.length];
+      }
+      return target[index];
```

Comment:
I see a problem here. As in what if the `Number(prop)` can not be converted to a number ?
In my opinion something like following could have been done.

`if (Number.isNaN(index)) {`
 `   return Reflect.get(target, property, receiver);`
   `}`
   `return target[index < 0 ? target.length + index : index];`